app.json

```diff
@@ -5,15 +5,22 @@
     "privacy": "public",
     "sdkVersion": "32.0.0",
     "platforms": ["ios", "android"],
-    "icon": "./appIcon.png",
+    "icon": "./appIcon-android.png",
     "packagerOpts": {
       "nonPersistent": true
     },
     "updates": {
       "fallbackToCacheTimeout": 0
     },
+    "ios": {
+      "bundleIdentifier": "org.serlo.abc",
+      "icon": "./appIcon-ios.png",
+      "supportsTablet": true,
+      "buildNumber": "1.2.0"
+    },
     "android": {
       "package": "org.serlo.abc",
+      "icon": "./appIcon-android.png",
```

Comment:
That's fine, but we'll probably should use the iOS appIcon as default:
> Expo Client and Web
> If your app contains privacy: public in app.json, it will show up on your expo.io profile. We will mask your icon to have rounded corners in that circumstance, so if it already looks reasonable on iOS, it will probably look good here as well.

Reply:
:+1: ok